tools/ports/freetype.py

```diff
@@ -77,31 +76,23 @@ def create(final):
             'src/type1/type1.c',
             'src/type42/type42.c',
             'src/winfonts/winfnt.c']
-    commands = []
-    o_s = []
-    for src in srcs:
-      o = os.path.join(dest_path, shared.replace_suffix(src, '.o'))
-      shared.safe_ensure_dirs(os.path.dirname(o))
-      commands.append([shared.EMCC, '-c', os.path.join(dest_path, src), '-o', o,
-                       '-DFT2_BUILD_LIBRARY', '-O2',
-                       '-I' + dest_path + '/include',
-                       '-I' + dest_path + '/truetype',
-                       '-I' + dest_path + '/sfnt',
-                       '-I' + dest_path + '/autofit',
-                       '-I' + dest_path + '/smooth',
-                       '-I' + dest_path + '/raster',
-                       '-I' + dest_path + '/psaux',
-                       '-I' + dest_path + '/psnames',
-                       '-I' + dest_path + '/truetype',
-                       '-w',
```

Comment:
I assume it's intentional that you dropped -w here?

Reply:
Yes, build_port takes case or adding `-O2` and `-w`,